### PR TITLE
feat(gallery): Adding component -FRONT-1776

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/implementations/twig/components/gallery/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@ecl/twig-component-gallery",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "3.0.0",
+  "description": "ECL Gallery",
+  "private": true,
+  "publishConfig": {
```
Comment (emeryro) [inline src/implementations/twig/components/gallery/package.json]: this should be removed, as the package is set to private

File: src/implementations/vanilla/components/gallery/gallery.scss [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+
+.ecl-gallery__slider {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  position: relative;
+}
+
+.ecl-gallery__slider-image-container, // DEPRECATED
```
Comment (emeryro) [inline src/implementations/vanilla/components/gallery/gallery.scss]: the deprecated rule can probably be removed

File: src/implementations/vanilla/components/gallery/gallery.scss [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+}
+
+.ecl-gallery__footer {
+  border-top: 1px solid map.get(color.$color, 'grey-10');
+  margin-top: map.get(spacing.$spacing, 'xs');
+  padding-top: map.get(spacing.$spacing, 'm');
+}
+
+/* stylelint-disable-next-line order/order */
+@include breakpoints.up('md') {
```
Comment (emeryro) [inline src/implementations/vanilla/components/gallery/gallery.scss]:
```suggestion
@include breakpoints.up('m') {
```

File: src/implementations/vanilla/components/gallery/gallery.scss [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+}
+
+/* stylelint-disable-next-line order/order */
+@include breakpoints.up('m') {
+  .ecl-gallery__slider {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .ecl-gallery__slider-image-container, // DEPRECATED
```
Comment (emeryro) [inline src/implementations/vanilla/components/gallery/gallery.scss]: same here

**Review by emeryro — CHANGES_REQUESTED** [review]

**Comment by kalinchernev** [comment]
No issues after a check on my side. Only the need to update the branch to the latest version of next-v3.

**Pull request merged**